packages/ensnode-utils/src/subname-helpers.ts

```diff
@@ -17,6 +21,65 @@ export const ROOT_NODE = namehash("");
 
 export const makeSubnodeNamehash = (node: Hex, label: Hex) => keccak256(concat([node, label]));
 
+// normalize address to match the reverse address format
+// as per https://docs.ens.domains/resolution/names#reverse-nodes
+const normalizedAddressDigits = (address: Hex): string => address.slice(2).toLowerCase();
+
+export interface LabelByReverseAddressArgs {
+  /** The reverse address */
+  reverseAddress: Hex;
```

Comment:
```suggestion
  maybeReverseAddress: Hex;
```

Suggesting this to highlight how this maybe is the reverse address, but it's not guaranteed.